### PR TITLE
Add Anolis OS 7 and Anolis OS 8 templates and configs

### PR DESCRIPTION
**Pull request by mskdxl**

### CODE REVIEW

File: mock-core-configs/etc/mock/templates/anolis-7.tpl [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+protected_packages=
+user_agent={{ user_agent }}
+
+# repos
+
+[os]
+name=AnolisOS-7 - os
+baseurl=http://mirrors.openanolis.cn/anolis/7/os/$basearch/os
+enabled=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-ANOLIS
```
Comment (xsuchy) [inline mock-core-configs/etc/mock/templates/anolis-7.tpl:34]: Please add the key to https://github.com/xsuchy/distribution-gpg-keys/ and then use the path in the form of file:///usr/share/distribution-gpg-keys/...

Comment (xsuchy) [reply mock-core-configs/etc/mock/templates/anolis-7.tpl:34]: And please bump up mock-core-configs.spec to requires distribution-gpg-keys >= 1.74

File: mock-core-configs/etc/mock/templates/anolis-8.tpl [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+baseurl=http://mirrors.openanolis.cn/anolis/$releasever/AppStream/$basearch/os
+enabled=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-ANOLIS
+gpgcheck=1
+
+[PowerTools]
+name=AnolisOS-$releasever - PowerTools
+baseurl=http://mirrors.openanolis.cn/anolis/$releasever/PowerTools/$basearch/os
+enabled=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-ANOLIS
```
Comment (praiskup) [inline mock-core-configs/etc/mock/templates/anolis-8.tpl:45]: The gpg key location is going to be `file:///usr/share/distribution-gpg-keys/anolis/RPM-GPG-KEY-ANOLIS`

**Comment by praiskup** [comment]
Otherwise +1

File: mock-core-configs/etc/mock/templates/anolis-8.tpl (hunk 2 above)
Comment (praiskup) [reply mock-core-configs/etc/mock/templates/anolis-8.tpl:45]: I fixed myself, thank you for the PR.

**Pull request closed without merging**